src/query/atomWithQuery.ts

```diff
@@ -150,7 +177,7 @@ export function atomWithQuery<
           setData(result.data)
         }
       }
-      const defaultedOptions = queryClient.defaultQueryObserverOptions(options)
+      const defaultedOptions = queryClient.defaultQueryOptions(options)
```

Comment:
Is this supported in RQ v3 too?

Reply:
yep, also supported with RQ v3

Reply:
Thanks. Alright, I think I should include this change in #1239 (or something different), so that we can run tests with RQ v3.